getting_started/getting_started.py

```diff
@@ -119,17 +121,17 @@ def lf_textblob_polarity(x):
 
 # %% [markdown]
 # And much more!
-# For many more types of labeling functions–including over data modalities beyond text–see the other [tutorials](#) and [real-world examples](#).
+# For many more types of labeling functions — including over data modalities beyond text — see the other [tutorials](#) and [real-world examples](#).
 #
 # In general the process of developing labeling functions is, like any other development process, an iterative one that takes time.
 # However, in many cases it can be _orders-of-magnitude_ faster that hand-labeling training data.
-# For more detail on the process of developing labeling functions and other training data operators in Snorkel, see the [Introductory tutorial](#).
+# For more detail on the process of developing labeling functions and other training data operators in Snorkel, see the [Spam LFs tutorial](http://snorkel.org/use-cases/01-spam-tutorial).
```

Comment:
This should just point to spam somehow, not the LFs tutorial specifically (because it says developing labeling functions and other data operators in Snorkel)

Reply:
+1 This is a good motivation for getting the tutorial readme pages done?